### ISSUE TITLE
AAGI Colour Names Should Align

### ISSUE THREAD

**Issue by adamhsparks** [AAGI-Org-AU-Public/AAGITemplates#16]
I noted that @RussellAndrewEdson had colour names in the LaTeX template, we should be sure that the colour names are aligned across the packages/templates. I've opened an issue here as well, https://github.com/AAGI-Org-AU-Public/AAGIPalettes/issues/1

**Comment by RussellAndrewEdson** [comment]
Absolutely, and I'm happy to use the AAGIPalettes ones you've got there. I'll make the changes when I get a sec.
